corehq/motech/openmrs/repeater_helpers.py

```diff
@@ -84,10 +84,15 @@ def get_case_location_ancestor_repeaters(case):
     return []
 
 
-def get_openmrs_location_uuid(domain, case_id):
+def get_ancestor_location_openmrs_uuid(domain, case_id):
     case = CaseAccessors(domain).get_case(case_id)
-    location = get_case_location(case)
-    return location.metadata.get(LOCATION_OPENMRS_UUID) if location else None
+    case_location = get_case_location(case)
+    if not case_location:
+        return None
+    for location in reversed(case_location.get_ancestors(include_self=True)):
```

Comment:
why do you need to reverse here?
If its actually needed, would "ascending=True" help?

Reply:
We reverse because the locations are returned top-down. We need to find the first ancestor (bottom-up) with an OpenMRS UUID, otherwise we could have a situation where an Event is tagged with a Country instead of a Clinic, for example.

I think if we have more than one usecase that needs to reverse the order of the ancestors, then `ascending=True` would definitely be useful. I'll check.